src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -334,7 +334,8 @@ void AttitudeEstimatorQ::task_main()
 
 		} else if (ret == 0) {
 			// Poll timeout, do nothing
-			PX4_WARN("Q POLL TIMEOUT");
+// TODO: RobD debugging - quiet this for now
```

Comment:
This suggests a substantial issue with the driver - I can't accept this diff if this is really occurring.